includes/reports/overview/list-table/class-sensei-reports-overview-list-table-students.php

```diff
@@ -41,26 +41,31 @@ public function __construct( Sensei_Grading $grading, Sensei_Reports_Overview_Da
 	 * @return array The array of columns to use with the table
 	 */
 	public function get_columns() {
+
+		$this->get_all_students_ids();
```

Comment:
Is this a leftover?

Reply:
jup, [here](https://github.com/Automattic/sensei/pull/5091/commits/a112091dd113cbe839dfdf1d7fda1653420036c4)__